src/interpreter/plugin/HyperlinkPlugin.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license
+ * Copyright (c) 2023 Handsoncode. All rights reserved.
+ */
+
+import {FunctionArgumentType, FunctionPlugin, FunctionPluginTypecheck} from './FunctionPlugin'
+import {InterpreterState} from '../InterpreterState'
+import {InterpreterValue} from '../InterpreterValue'
+import {ProcedureAst} from '../../parser'
+
+export class HyperlinkPlugin extends FunctionPlugin implements FunctionPluginTypecheck<HyperlinkPlugin> {
+  public static implementedFunctions = {
```

Comment:
```suggestion
  public static implementedFunctions: ImplementedFunctions = {
```

Reply:
Heh. Who came up with this? 😄 

Added ImplementedFunctions inheritance.